proposed/http-factory/http-factory.md

```diff
@@ -163,14 +170,17 @@ fwrite($resource, $body);
 Has the ability to create streams for uploaded files.
 
 ```php
-namespace Psr\Http\Message;
+namespace Psr\Http\Factory;
+
+use Psr\Http\Message\UploadedFileInterface;
 
 interface UploadedFileFactoryInterface
 {
     /**
      * Create a new uploaded file.
      *
-     * If a string is passed it is assumed to be a file path.
+     * If a string is used to create the file, a temporary resource will be
+     * created with the content of the string.
```

Comment:
@weierophinney perhaps a better question is... in a system that does not have `$_FILES` compatibility, why would creating uploaded file instances be necessary at all? Why wouldn't those systems simply use the existing stream to write a file or process in some other way?